certified-connectors/PowerToolsLITE/apiDefinition.swagger.json

```diff
@@ -1818,154 +3510,693 @@
       ],
       "properties": {
         "input": {
-          "type": "array",
-          "description": "Colllection of values to calculate",
-          "x-ms-summary": "Values",
-          "items": {
-            "type": "number"
-          }
+          "type": "array",
+          "description": "Colllection of values to calculate",
```

Comment:
@eshupps please make sure that everything is grammatically correct and is free of typographical errors. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
Will fix the typo and resubmit.